Assets/Code/src/Runtime/Match/MatchState.cs

```diff
@@ -7,6 +7,8 @@
 
 namespace HouraiTeahouse.FantasyCrescendo.Matches {
 
+public enum MatchStateID { Intro, InGame, Pause, End }
```

Comment:
Document this and each of the enum values.

Also not sure about the naming of MatchStateID, doesn't seem right, but MatchState is already being used. MatchProgressionStage?